packages/office-addin-cli/package.json

```diff
@@ -37,6 +37,7 @@
     "eslint-plugin-prettier": "^3.4.0",
     "mocha": "^8.3.2",
     "office-addin-prettier-config": "^1.0.16",
+    "prettier": "^2.1.2",
```

Comment:
This dependency on prettier shouldn't be needed, right?  Shouldn't it be be office-addin-lint?

Reply:
office-addin-lint depends on office-addin-cli so a dependency the other way around would create a circular dependency.  This is a way to get the linting/prettier stuff in the project w/out the dependency on office-addin-lint.